src/test.rs

```diff
@@ -15,6 +15,15 @@ use crate::{
     source_code::{Indexer, Locator, Stylist},
 };
 
+pub fn test_resource_path(path: impl AsRef<std::path::Path>) -> std::path::PathBuf {
+    std::path::Path::new(
+        // We intentionally don't use Path::join to append path since that uses
+        // a \ as the path separator on Windows causing a fixture mismatch for
+        // flake8_2020, resulting in a CI failure for `cargo test` on Windows.
+        format!("./resources/test/{path}"),
```

Comment:
While stumbling into this I added the now 1st commit of this PR since the GitHub CI previously with `cargo insta test` didn't print the diff.

Reply:
Nevermind, I found a better way, see the last commit.